meetups/2024/2024-02-22-talks/program.md

```diff
@@ -0,0 +1,50 @@
+# Rust Aarhus meetup at Partisia
+
+Date: 2024-02-22
+
+Rust Aarhus invites you to this free meetup for all interested in the programming language [Rust].
+
+The program for the evening is:
+
+- 18:05 - 18:10: Welcome (5 min)
+- 18:15 - 19:00: "Building serverless apps, using WASI, WebAssembly components using Rust" by Mikkel Mørk Hegnhøj (45 min incl Q&A)
+- 19:00 - 19:10: A word from our sponsor (5-10 min)
+- 19:10 - 19:55: Light dinner sponsored by Partisia (45 min)
+- 19:55 - 20:40: "Zero-copy deserialization in Rust" by Gustav Wengel (45 min incl Q&A)
+- 20:45 - 20:55: Book Lottery (10 min)
+- 20:55 - 21:00: Closing remarks (5 min)
+- 21:00: Let's continue the discussion at the nearby bar [INSERT BAR NAME].
```

Comment:
Lets omit this part about the bar for now, so we can get this published, then we can always update it later to add the bar.
```suggestion
```